libraries/AP_HAL_ChibiOS/hwdef/FlywooF745/hwdef.dat

```diff
@@ -0,0 +1,158 @@
+define HAL_BUZZER_PIN 80
+define HAL_BUZZER_ON 1
+define HAL_BUZZER_OFF 0
+
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# Debug
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# SPI1 for Flash Storage 16mb
+PA4 FLASH_CS CS
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2 for OSD
+PB12 AT7456E_CS CS
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI4 for MPU6000
+PE4 MPU6000_CS CS
+PE2 SPI4_SCK SPI4
+PE5 SPI4_MISO SPI4
+PE6 SPI4_MOSI SPI4
+
+# I2C1 for baro
+PB6 I2C1_SCL I2C1 PULLUP
+PB7 I2C1_SDA I2C1 PULLUP
+
+PC3 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PC2 BATT_CURRENT_SENS ADC1 SCALE(1)
+
+# define default battery setup
+define HAL_BATT_VOLT_PIN 13
+define HAL_BATT_CURR_PIN 12
+define HAL_BATT_VOLT_SCALE 10.9
+define HAL_BATT_CURR_SCALE 28.5
+define HAL_BATT_MONITOR_DEFAULT 4
+
+PC5 RSSI_ADC ADC1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6 UART7
+
+# USART1
+PA10 USART1_RX USART1
+PA9  USART1_TX USART1
+
+# USART2
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2 NODMA
+
+# USART3
+PB10 USART3_TX USART3
+PB11 USART3_RX USART3 NODMA
+
+# UART4
+PA0 UART4_TX UART4
+PA1 UART4_RX UART4 NODMA
+
+# UART5
+PC12 UART5_TX UART5
+PD2  UART5_RX UART5 NODMA
+
+# USART6
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6 NODMA
+
+# UART7
+PE7 UART7_RX UART7 NODMA
+PE8 UART7_TX UART7 NODMA
+
+# Motors
+
+PB0  TIM3_CH3 TIM3 PWM(1) GPIO(50) 			# Motor 1
+PB1  TIM3_CH4 TIM3 PWM(2) GPIO(51) BIDIR	# Motor 2
+PE9  TIM1_CH1 TIM1 PWM(3) GPIO(52) 			# Motor 3
+PE11 TIM1_CH2 TIM1 PWM(4) GPIO(53) BIDIR	# Motor 4
+PC9  TIM8_CH4 TIM8 PWM(5) GPIO(54) BIDIR	# Motor 5
+PA3  TIM5_CH4 TIM5 PWM(6) GPIO(55) BIDIR	# Motor 6
+PB4  TIM3_CH1 TIM3 PWM(7) GPIO(56)			# Motor 7
+PB5  TIM3_CH2 TIM3 PWM(8) GPIO(57) BIDIR	# Motor 8
+
+
+
+# NeoPixel LED strip
+PD12 TIM4_CH1 TIM4 PWM(9) GPIO(58)
+PA2 LED0 OUTPUT LOW GPIO(90) # LED
+
+
+# Misc Timer Stuff
+#PE13 TIM1_CH3 TIM1 PWM(10) GPIO(59)			# PPM/Unused - for future use according to Flywoo
+#PB3  TIM2_CH2 TIM2 PWM(11) GPIO(60)			# Unused - for future use according to Flywoo
+
+
+DMA_PRIORITY USART1* TIM1* TIM3*
+DMA_NOSHARE SPI1_TX TIM1_CH1 TIM3_CH3
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+# enable logging to dataflash
+define HAL_LOGGING_DATAFLASH
+
+# spi devices
+SPIDEV dataflash SPI1 DEVID1 FLASH_CS   MODE3 32*MHZ 32*MHZ
+SPIDEV osd SPI2 DEVID2 AT7456E_CS MODE0 10*MHZ 10*MHZ
+SPIDEV mpu6000 SPI4 DEVID4 MPU6000_CS MODE3  1*MHZ  4*MHZ
+
+# no built-in compass and no external I2C so no compass
+define ALLOW_ARM_NO_COMPASS
+define HAL_COMPASS_DEFAULT HAL_COMPASS_NONE
+define HAL_I2C_INTERNAL_MASK 0
+
+# one IMU
+IMU Invensense SPI:mpu6000 ROTATION_NONE
+define HAL_DEFAULT_INS_FAST_SAMPLE 1
+
+# one BARO
+BARO BMP280 I2C:0:0x76
+
+# setup for OSD
+define OSD_ENABLED 1
+define HAL_OSD_TYPE_DEFAULT 1
+ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
+
+# FFT option (disabled by default)
+# define BOARD_PWM_COUNT_DEFAULT 7
```

Comment:
Is this correct? You have 9 PWM channels above

Reply:
This was a mistake. It should not be commented out, and you're right 9 PWM not 7. Fixing/committing shortly.